packages/cfd/src/Containers/mt5-desktop-redirect-option.tsx

```diff
@@ -8,7 +8,15 @@ import './mt5-mobile-redirect-option.scss';
 
 type TMT5DesktopRedirectOptionProps = {
     account_title: string;
-    mt5_trade_account: DetailsOfEachMT5Loginid;
+    mt5_trade_account: DetailsOfEachMT5Loginid & {
```

Comment:
i see we are updating the types in multiple places. Can we add a common type in `packages/cfd/src/types/types.ts` and reuse it. i believe this will anyway get removed when api-types is updated with new schema.

Reply:
consolidated redundant type definition of `TDetailsOfEachMT5Loginid`